flow/Makefile

```diff
@@ -361,7 +361,9 @@ $(OBJECTS_DIR)/klayout_tech.lef: $(TECH_LEF)
 	sed '/OR_DEFAULT/d' $< > $@
 
 $(OBJECTS_DIR)/klayout.lyt: $(KLAYOUT_TECH_FILE) $(OBJECTS_DIR)/klayout_tech.lef
-	sed 's,<lef-files>.*</lef-files>,$(foreach file, $(OBJECTS_DIR)/klayout_tech.lef $(SC_LEF) $(ADDITIONAL_LEFS),<lef-files>$(shell realpath --relative-to=$(RESULTS_DIR) $(file))</lef-files>),g' $< > $@
+	SC_LEF_RELATIVE_PATH="$$\(env('FLOW_HOME')\)/$(shell realpath --relative-to=$(FLOW_HOME) $(SC_LEF))"; \
```

Comment:
@rovinski You mentioned somewhere, can't find it now, that with relative paths this preprocessing was no longer necessary. However, this preprocessing includes other files than just the one that is in [asap7.lyt](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/1c4290036b48bd71c4eb05a9fe48951a87ba3ab1/flow/platforms/asap7/KLayout/asap7.lyt#L50)

Did you mean that we could remove `$(OBJECTS_DIR)/klayout_tech.lef` below and just leave in whatever is in `asap7.lyt` already?

Reply:
@oharboe that's a good point. It seems that that `klayout_tech.lyt` only exists because KLayout could not properly parse some custom `LEF PROPERTY` syntax used. Specifically, marking certain vias as `OR_DEFAULT` so that GRT and DRT know which via to use, IIRC. The current solution is to simply create a copy in `$(OBJECT_DIR)` but trim out the `OR_DEFAULT` lines.

A better solution to this issue is to see if a recent version of KLayout handles this LEF syntax now, and if so, require that version. Another good solution would be to reexamine if marking vias as `OR_DEFAULT` is still required or if that is obsolete (@maliberty?)